gnu-apl-interactive.el

```diff
@@ -117,9 +168,7 @@ code was read from."
 (defun gnu-apl--erase-and-set-function (name _content)
   (gnu-apl-interactive-send-string (concat "'" *gnu-apl-ignore-start* "'"))
   (gnu-apl-interactive-send-string (concat ")ERASE " name))
-  (gnu-apl-interactive-send-string (concat "'" *gnu-apl-ignore-end* "'"))
-  (setq gnu-apl-function-content-lines (split-string content "\n"))
-  (gnu-apl-interactive-send-string (concat "'" *gnu-apl-send-content-start* "'")))
+  (gnu-apl-interactive-send-string (concat "'" *gnu-apl-ignore-end* "'")))
```

Comment:
This seems to be a mistake. The code that updates the function is gone.

Reply:
That might be the case, but I don't see where `*gnu-apl-send-content-start* ` is used.

Reply:
I wrote that code a long time ago, and I believe that the entire function isn't needed anymore. Its job is now handled by `gnu-apl--send-si-and-send-new-function`.

Reply:
You are right, the function is dead code. I'll remove it in that case.